website/source/api/agent.html.md

```diff
@@ -249,6 +249,112 @@ $ curl \
     https://consul.rocks/v1/agent/maintenance?enable=true&reason=For+API+docs
 ```
 
+## View Metrics
+
+This endpoint returns the configuration and member information of the local
+agent.
+
+| Method | Path                         | Produces                   |
+| ------ | ---------------------------- | -------------------------- |
+| `GET`  | `/agent/metrics`             | `application/json`         |
+
+This endpoint will dump the metrics for the most recent finished interval.
+For more information about metrics, see the [telemetry](/docs/agent/telemetry.html)
+page.
+
+| Blocking Queries | Consistency Modes | ACL Required |
+| ---------------- | ----------------- | ------------ |
+| `NO`             | `none`            | `agent:read` |
+
+### Sample Request
+
+```text
+$ curl \
+    https://consul.rocks/v1/agent/metrics
+```
+
+### Sample Response
+
+```json
+{
+    "Timestamp": "2017-08-08 02:55:10 +0000 UTC",
```

Comment:
It's for the interval the metrics were collected in - if a different timestamp comes back, it's a different set of metrics. I'll add the info about the fields below this example